src/components/AssetsSearch/AssetsSearch.scss

```diff
@@ -0,0 +1,9 @@
+.form-inline label {
+  margin-right: 10px;
+  font-weight: 600;
```

Comment:
Can we use [$font-weight-bold]($font-weight-bold) here? Doesn't everyone love theming?

Reply:
I guess alternatively one could extend the `font-weight-bold` css class too--seems like overkill perhaps though.

Reply:
I tried that but it is set to 700 and I think it looks too "heavy". What do you think?
![screenshot-20180226174240-472x84](https://user-images.githubusercontent.com/1505923/36700233-185c6676-1b1d-11e8-9232-72afe6cd3a13.png)

vs. what it is set to now (600):
![screenshot-20180226174820-461x89](https://user-images.githubusercontent.com/1505923/36700292-40c389a0-1b1d-11e8-97eb-c7ba30405267.png)

Reply:
Yeah, I see what you mean! @abbyluke, do you have an opinion on this? Is the [edx-bootstrap bold](https://github.com/edx/edx-bootstrap/blob/f0a6237174de7dd65815b07c73da127f082faa4d/sass/edx/_variables.scss#L237) in this case too bold?